Rakefile

```diff
@@ -120,3 +128,48 @@ namespace :qless do
   end
 end
 
+
+if ruby_meet_expectation("2.0")
```

Comment:
Also, for this kind of thing I usually just try requiring the gem and then `rescue LoadError`, rather than checking the ruby version.  For example:

https://github.com/seomoz/vanguard/blob/3acfb1b2ae63da8a294b34f1e170ae2283f8df26/tasks/specs.rb#L1-L7

I think this is better because the ruby version check duplicates the knowledge of what versions cane should be available on, embedding it both in the Gemfile and in the Rakefile.   If/when that changes, you'd have to change in two places.  With the require-and-rescue-load-error pattern, it's only in the Gemfile, and here in the rakefile it simply uses cane if available.